.github/workflows/ci-schedule-compatibility.yaml

```diff
@@ -0,0 +1,66 @@
+name: CI Schedule Compatibility Testing Workflow
+on:
+  schedule:
+    # Run this workflow "At 18:00 UTC on Sunday and Saturday"
+    - cron: '0 18 * * 0,6'
```

Comment:
How about separating this time from `CI Schedule Workflow`?

Reply:
How about `0 20 * * 0,6`?

Reply:
cc @XiShanYongYe-Chang

Reply:
It's ok to me~